documentation/install_from_packages.markdown

```diff
@@ -26,20 +24,27 @@ Puppet provides official packages that install Puppet Server 2.4 and all of its
 
 ### Debian
 
--   Debian 7 (Wheezy)
 -   Debian 8 (Jessie)
 
+Java 8 runtime packages do not exist in the standard repositories for Debian 8 (Jessie). In order to install Puppet Server for Jessie, users would need to first configure the jessie-backports repository (see: https://backports.debian.org/Instructions/), which does have openjdk-8. The following steps should install Puppet Server properly for Jessie:
+
+~~~
+echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d
+apt-get update
+apt-get -t jessie-backports install "openjdk-8-jdk-headless"
+apt-get install puppetserver
+~~~~
+
 ### Ubuntu
 
--   Ubuntu 12.04 (Precise)
--   Ubuntu 14.04 (Trusty)
--   Ubuntu 15.10 (Wily)
 -   Ubuntu 16.04 (Xenial)
 
 ### SUSE Linux Enterprise Server
 
 -   SLES 12
 
+Users will need to be running SP1 or later in order to be able to install openjdk8 properly for SLES 12.
```

Comment:
Can this change simply add SP1 to the SLES 12 bullet point without the additional detail? If not, can we describe the steps users need to take to install the package on SLES 12, or at least describe the challenges involved in more detail?